awx.yaml

```diff
@@ -0,0 +1,89 @@
+package:
+  name: awx
+  version: 24.6.1
+  epoch: 0
+  description: AWX provides a web-based user interface, REST API, and task engine built on top of Ansible
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - python-3.11
+
+# To do- add py-twilio and py-zstd and hiredis-py
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - bash
+      - make
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+      - nvm
+      - py3-sqlparse
+      - py3.11-pip
+      - python-3.11-dev
+      - zlib-dev
+      - py3-aiohttp
+      - py3-cffi
+      - py3-charset-normalizer
+      - py3-pydantic-core
+      - py3-cryptography
+      - py3-cython
+      - py3-frozenlist
+      - py3-grpcio
+      - py3-lxml
+      - py3-markupsafe
+      - py3-maturin
+      - py3-msgpack
+      - py3-multidict
+      - py3-protobuf
+      - py3-psutil
+      - py3-psycopg
+      - py3-pycparser
+      - py3-rpds-py
+      - py3-wrapt
+      - py3-yarl
+      - py3-pyyaml
+      - py3-zope.interface
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ansible/awx.git
+      tag: ${{package.version}}
+      expected-commit: 94e5795dfc37b95c576d61f3e3b4e936c021548c
+
+  - runs: |
+      # Back-end build
+
+      mkdir -p ${{targets.contextdir}}/var/lib/awx
+      python -m venv --system-site-packages  ${{targets.contextdir}}/var/lib/awx/venv
+      export VENV_BASE=${{targets.contextdir}} make requirements_awx
```

Comment:
This needs to be       `VENV_BASE=${{targets.contextdir}}/var/lib/awx make requirements_awx` 

I would try without the py3.11 packages first to get the list of packages then validate the pinned constraints vs versions we provide